config/7.2.0/securebanking/ig/routes/routes-service/200-events-admin-api.json

```diff
@@ -0,0 +1,148 @@
+{
+  "comment": "Handle Open Banking Domestic Payment request",
+  "name": "200 - Open Banking events admin API",
+  "auditService": "AuditService-OB-Route",
+  "baseURI": "${urls.rsBaseUri}",
+  "condition": "${matches(request.uri.path, '^/rs/admin/data/events')}",
+  "capture": [
+    "response",
+    "request"
+  ],
+  "handler": {
+    "type": "Chain",
+    "config": {
+      "filters": [
+        "SBATFapiInteractionFilterChain",
+        {
+          "comment": "Sign event notification from the request",
+          "name": "SignEvents",
+          "type": "ScriptableFilter",
+          "config": {
+            "type": "application/x-groovy",
+            "file": "SignEventNotifications.groovy",
+            "args": {
+              "routeArgAlgorithm": "PS256",
+              "routeArgSecretId": "jwt.signer",
+              "routeArgKid": "&{ig.ob.aspsp.signing.kid}",
+              "routeArgSecretsProvider": "${heap['SecretsProvider-ASPSP']}",
+              "routeArgTrustedAnchor": "openbanking.org.uk",
+              "obAspspOrgId": "&{ob.aspsp.org.id}"
+            }
+          }
+        },
+        {
+          "comment": "Ensure the response is Open Banking compliant",
+          "name": "ObResponseCheck",
+          "type": "ScriptableFilter",
+          "config": {
+            "type": "application/x-groovy",
+            "file": "ObResponseCheck.groovy"
+          }
+        },
+        {
+          "comment": "Extract client certificate thumbprint for cert bound access tokens",
+          "name": "CertificateThumbprintFilter-1",
+          "type": "CertificateThumbprintFilter",
+          "config": {
+            "certificate": "${pemCertificate(urlDecode(request.headers['ssl-client-cert'][0]))}",
+            "failureHandler": {
+              "type": "ScriptableHandler",
+              "config": {
+                "type": "application/x-groovy",
+                "file": "ReturnInvalidCnfKeyError.groovy"
+              }
+            }
+          }
+        },
+        {
+          "comment": "Check certificate bound access token",
+          "name": "OAuth2ResourceServerFilter-OB",
+          "type": "OAuth2ResourceServerFilter",
+          "config": {
+            "scopes": [
+              "accounts",
+              "payments",
+              "fundsconfirmations",
+              "openid"
+            ],
```

Comment:
The access token must have all scopes to perform operations against the admin data event API